src/Commands/InvalidateCacheCommand.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace Rapidez\Statamic\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Cache;
+use Illuminate\Support\Facades\DB;
+use Rapidez\Core\Facades\Rapidez;
+use Statamic\StaticCaching\Cacher;
+
+class InvalidateCacheCommand extends Command
+{
+    protected $signature = 'rapidez-statamic:invalidate-cache';
+
+    protected $description = 'Search for invalidatable caches';
+
+    public $urls;
+
+    public $latestCheck;
+
+    public function handle(Cacher $cacher): void
+    {
+        $this->latestCheck = Cache::get($this->signature);
+
+        if (!$this->latestCheck) {
+            $this->info('Cleared all urls (as we do not have a latest check date yet)');
+            $this->setLatestCheckDate();
+            $cacher->flush();
+            return;
+        }
+
+        $this->setLatestCheckDate();
+        $stores = Rapidez::getStores();
+
+        foreach ($stores as $store) {
+            Rapidez::setStore($store);
+
+            $this->urls = collect();
+
+            $this
+                ->addProductsUrls()
+                ->addCategoryUrls()
+                ->addPageUrls()
+                ->makeAbsolute();
```

Comment:
Isn't this just invalidating all the urls too? Why not do `$cacher->flush();` here too then? Or am i missing something?